payments/api/payments.api

```diff
@@ -0,0 +1,7 @@
+public final class com/stripe/android/payments/BuildConfig {
```

Comment:
This is unchanged from previous, but the new version enforces this for modules with only generated code now.